contracts/interfaces/modules/royalty/policies/IIpRoyaltyVault.sol

```diff
@@ -14,6 +14,12 @@ interface IIpRoyaltyVault {
     /// @param unclaimedTokens The amount of unclaimed tokens at the snapshot
     event SnapshotCompleted(uint256 snapshotId, uint256 snapshotTimestamp, uint32 unclaimedTokens);
 
+    /// @notice Event emitted when revenue tokens are claimed
+    /// @param claimer The address of the claimer
+    /// @param token The revenue token claimed
+    /// @param amount The amount of revenue token claimed
+    event RevenueTokensClaimed(address claimer, address token, uint256 amount);
```

Comment:
Claimer and token should probably be `indexed` for easier event filtering

Reply:
Also, RevenueToken"s"Claimed or RevenueTokenClaimed? I think the latter might be better since it's just one token?

Reply:
SDK team said they don't want this